topdown/time.go

```diff
@@ -82,6 +82,17 @@ func builtinParseDurationNanos(_ BuiltinContext, operands []*ast.Term, iter func
 	return iter(ast.NumberTerm(int64ToJSONNumber(int64(value))))
 }
 
+func builtinTimestamp(_ BuiltinContext, operands []*ast.Term, iter func(*ast.Term) error) error {
+	t, err := tzTime(operands[0].Value)
+	if err != nil {
+		return err
+	}
+	// Using RFC3339Nano time formatting
+	layout := time.RFC3339Nano
```

Comment:
It seems to me that it would be sensible to allow the user to provide the layout here. I can imagine that e.g. the nano seconds part is considered too granular in most cases, and in other cases some other format altogheter is likely desired. This looks like a good opportunity to provide that type of flexibility, or what do you think?

Reply:
hey @anderseknert, I can add the layout option in the same PR as it makes sense to me too. I just have one concern regarding that. Can you address the second point [here](https://github.com/open-policy-agent/opa/pull/5432#issuecomment-1335297862) for clarity?

Reply:
I've updated the PR with the layout format feature for now